end-to-end-test/remote/specs/core/results.spec.js

```diff
@@ -467,8 +467,8 @@ describe('Results Page', function() {
                 browser.isVisible(`${noBannerSelector}.plots-oql-status-banner`)
             );
         });
-        it('should be present in mutations tab with explicit query', function() {
-            browser.click('a.tabAnchor_mutations');
+        it('should be present in alterations tab with explicit query', function() {
+            browser.click('a.tabAnchor_alterations');
```

Comment:
@pvannierop couple of a problems here. 
1. the selector should be .comparison-oql-status-banner, not mutations...
2. It expects the unaffected banner, but we actually get the affected banner.   Probably because comparison has hereto-for had that unaffected banner.  So there's a question what SHOULD it be.  I think maybe it should be unaffected, which would require a code change and perhaps some re-thinking.  Will check with Ino and JJ what we should do.

Reply:
ok, i have this one solved.